app/portainer/components/forms/kubernetes-redeploy-app-git-form/kubernetes-redeploy-app-git-form.controller.js

```diff
@@ -39,9 +48,23 @@ class KubernetesAppGitFormController {
   async pullAndRedeployApplication() {
     return this.$async(async () => {
       try {
+        //Analytics
+        const metadata = {};
+
+        if (this.formValues.AutoUpdate.RepositoryAutomaticUpdates) {
+          if (this.formValues.AutoUpdate.RepositoryMechanism === `Interval`) {
```

Comment:
@fhanportainer I would recommend usage of constant over string for all references